package.json

```diff
@@ -34,7 +34,6 @@
   },
   "devDependencies": {
     "@vercel/ncc": "^0.34.0",
-    "eslint": "^7.32.0",
```

Comment:
Why do we drop eslint in this commit, to bring it back in in the next one?

Reply:
Ah, I can fix that to reduce noise. The linting and TS work were originally done kind of in tandem and I only started commit splitting when I was prepping the final patchset, and so I was somewhat reconstructing history from memory. I must have forgotten `eslint` wasn't net-new :)